tests/cpp/gbm/test_gblinear.cc

```diff
@@ -37,15 +37,15 @@ TEST(GBLinear, Json_IO) {
 
   model = Json::Load({model_str.c_str(), model_str.size()});
   ASSERT_TRUE(IsA<Object>(model));
-  model = model["model"];
 
   {
+    model = model["model"];
     auto weights = get<Array>(model["weights"]);
     ASSERT_EQ(weights.size(), 17);
   }
 
   {
-    model = Json::Load({model_str.c_str(), model_str.size()});
+    model = Json::Load(StringView{model_str.c_str(), model_str.size()});
```

Comment:
Still not sure why you are loading from `model_str` twice. I don't see `model_str` modified in the middle, so it's not a round-trip test.

Reply:
@hcho3 Thanks for catching it.  Removed in latest commit.  That's an oversight.  As for the string view,  I have a better implementation in #5046 .  Will consider merging some parts of that optimization PR so that we have better performance without scarifying the API.